pkg/fission-cli/cmd/function/log.go

```diff
@@ -60,13 +61,13 @@ func (opts *LogSubCommand) do(input cli.Input) error {
 		return errors.Wrap(err, "error getting function")
 	}
 
-	server, err := util.GetApplicationUrl(input.Context(), opts.Client(), "application=fission-api")
-	if err != nil {
-		return err
+	logDBOptions := logdb.LogDBOptions{
+		CTX:    input.Context(),
```

Comment:
Context we should not pass in struct I think. We can modify GetLogsDB so that it accepts context.
Function can use or ignore context.